<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMailItem</class>
 <widget class="QWidget" name="WidgetMailItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Mail</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxMain">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  border-radius: 0px;
  border: 0px solid rgb(131, 147, 167);
  background-color: rgb(80,112,125);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
          <item>
           <widget class="QLabel" name="labelDate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>&lt;Date&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Delete Mail</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-delete-small.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}
QPushButton:checked {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-notify.png);
}
QPushButton:checked:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelShortText">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelLine">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>clicked()</signal>
   <receiver>WidgetMailItem</receiver>
   <slot>onBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnEditClicked()</slot>
  <slot>onBtnNewLocationClicked()</slot>
  <slot>onBtnNotificationClicked()</slot>
  <slot>onBtnDeleteClicked()</slot>
 </slots>
</ui>
